egograph/pipelines/sources/browser_history/ingest_pipeline.py

```
@@ -73,6 +89,42 @@ def run_browser_history_pipeline(
                 )
         events_saved = True
 
+    # YouTube watch event 抽出・メタデータ解決・保存
+    youtube_events = extract_youtube_watch_events(rows)
+    if youtube_events:
+        # メタデータ解決（API クライアントが利用可能な場合）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)